docs/book/src/topics/provider-upgrades.md

```diff
@@ -0,0 +1,36 @@
+# Upgrading the Cluster API AWS Provider
+
+The `clusterctl` tool ships with an upgrade path designed to check for available updates and apply a contract to upgrade a CAPA management cluster to the latest version. The upgrade path will download manifests for the latest versions of Cluster API and the AWS provider, apply those to your management cluster and scale down the `Deployments` running older versions.
+
+During the upgrade, clusters provisioned by CAPA will continue to function normally. Still, the ability to use the management cluster to provision new clusters/manage or reconcile existing clusters will be restricted. For production workloads, it's best to upgrade your CAPA management cluster when you know you won't need to create or modify existing clusters.
```

Comment:
Slight change in wording as sounded strange to me. Very nitpick-y.
```suggestion
During the upgrade, clusters provisioned by CAPA will continue to function normally. Still, the ability to use the management cluster to provision new clusters or manage / reconcile existing clusters will be restricted. For production workloads, it's best to upgrade your CAPA management cluster when you know you won't need to create or modify existing clusters.
```

Reply:
Thannks for flagging this - pushed in a squashed commit with the tweak.